examples/SampleApp/src/screens/NewDirectMessagingScreen.tsx

```diff
@@ -222,24 +226,32 @@ export const NewDirectMessagingScreen: React.FC<NewDirectMessagingScreenProps> =
    *
    * 2. And then navigate to ChannelScreen
    */
-  const customSendMessage = async () => {
-    if (!currentChannel?.current) return;
+  const customSendMessage = async (
+    _: string,
+    message: StreamMessage,
+  ): Promise<SendMessageAPIResponse> => {
```

Comment:
You don't need the return type, without it it will infer the generics correctly too.